getting-started.md

```diff
@@ -7,7 +7,7 @@ Places requires special provisioning context with your regular Adobe Experience
 To get provisioned, complete the following tasks:
 
 * In the Admin Console, ensure that your organization is provisioned with Places.   If you are not the Adobe administrator, contact the administrator to ensure the Places context in Admin Console.
-* Ensure you are listed as a user under the Places and Launch product contexts in the Admin Console.  For more information, see [Adding a user to Launch and Places](https://placesdocs.com/places-services-by-adobe-documentation/adding-a-user-to-launch-and-places).
+* Ensure you are listed as a user under the Places and Launch product contexts in the Admin Console.  For more information, see [Adding a user to Launch and Places](adding-a-user-to-launch-and-places.md).
 * Ensure your mobile app property is configured in Experience Platform Launch with the Places extension installed.   For more information, see [Places extension](https://placesdocs.com/places-services-by-adobe-documentation/configure-places-in-the-sdk/places-extension). 
 * If your company has multiple Adobe organizations, ensure that you always select the organization that is provisioned for Places.  This allows you to add your POIs and Libraries.
```

Comment:
- Please replace "Places" with "Location Service".
- Please lower case the "L" in "libraries".